src/index.js

```diff
@@ -11,6 +11,11 @@ export const Root = styled.div(
   color
 )
 
+export const TypographyContext = createContext({})
```

Comment:
It *might* make sense to use Emotion's `ThemeContext` here https://github.com/emotion-js/emotion/blob/master/packages/core/src/index.js#L2